File: src/main/java/ru/stnk/RestTestAPI/entity/QuotationsScreenData.java
package ru.stnk.RestTestAPI.entity;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import java.util.Arrays;

@JsonIgnoreProperties(ignoreUnknown = true)
public class QuotationsScreenData {
    private QuotationsPairsData[] pairs_data;

    public QuotationsScreenData() {
    }

    public QuotationsPairsData[] getPairs_data() {
        return pairs_data;
    }

    public void setPairs_data(QuotationsPairsData[] pairs_data) {
        this.pairs_data = pairs_data;
    }

    @Override
    public String toString() {
        return "QuotationsScreenData{" +
                "pairs_data=" + Arrays.toString(pairs_data) +
                '}';
    }
}

File: src/main/kotlin/ru/stnk/resttestapi/entity/QuotationsScreenData.kt
package ru.stnk.resttestapi.entity

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.util.*

@JsonIgnoreProperties(ignoreUnknown = true)
class QuotationsScreenData {
    var pairs_data: Array<QuotationsPairsData>? = null

    override fun toString(): String {
        return "QuotationsScreenData{" +
                "pairs_data=" + Arrays.toString(pairs_data) +
                '}'.toString()
    }
}
